recipes/scala/src/main/scala/com/adobe/platform/ml/ScoringDataLoader.scala

```diff
@@ -0,0 +1,71 @@
+package com.adobe.platform.ml
+
+import com.adobe.platform.dataset.DataSetOptions
+import com.adobe.platform.ml.config.ConfigProperties
+import com.adobe.platform.ml.sdk.DataLoader
+import org.apache.spark.ml.feature.StringIndexer
+import org.apache.spark.sql.expressions.Window
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types.TimestampType
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+
+class ScoringDataLoader extends DataLoader {
+
+  override def load(configProperties: ConfigProperties, sparkSession: SparkSession): DataFrame = {
+
+    val serviceToken: String = sparkSession.sparkContext.getConf.get("ML_FRAMEWORK_IMS_ML_TOKEN", "").toString
+    val userToken: String = sparkSession.sparkContext.getConf.get("ML_FRAMEWORK_IMS_TOKEN", "").toString
+    val orgId: String = sparkSession.sparkContext.getConf.get("ML_FRAMEWORK_IMS_ORG_ID", "").toString
+
+    val dataSetId: String = configProperties.get("dataSetId").getOrElse("5c05ba9c7276150000be60f2")
+    val batchId: String = configProperties.get("batchId").getOrElse("e1df7c83c030498685ce4e5f5072c179")
+    val apiKey:String = configProperties.get("apiKey").getOrElse("acp_machineLearning_customer")
+
+    var df = sparkSession.read.format("com.adobe.platform.dataset")
+      .option(DataSetOptions.serviceToken, serviceToken)
+      .option(DataSetOptions.userToken, userToken)
+      .option(DataSetOptions.serviceApiKey, apiKey)
+      .option(DataSetOptions.orgId, orgId)
+      .option(DataSetOptions.batchId, batchId)
+      .load(dataSetId)
+
+    import sparkSession.implicits._
```

Comment:
Do imports all belong at top?

Reply:
This import cannot be outside since the variable sparkSession is accessible from within the load method.